File: solutions/2168-check-if-numbers-are-ascending-in-a-sentence/solution.java
class Solution {
    public boolean areNumbersAscending(String s) {
        List<Integer>ls=new ArrayList<>();
        String[] words = s.split("\\s+");
       for (String word : words) {
            try {
                int number = Integer.parseInt(word);
                ls.add(number);
            } catch (NumberFormatException e) {
                // Skip non-integer words
            }
        }
        for(int i=0;i<ls.size()-1;i++)
        {
            if(!(ls.get(i)<ls.get(i+1)))
                return false;
        }
        return true;
    }
}

File: solutions/2168-check-if-numbers-are-ascending-in-a-sentence/solution.py
class Solution(object):
    def areNumbersAscending(self, s):
        """
        :type s: str
        :rtype: bool
        """
        s=s.split()
        a=[int(i) for i in s if i.isdigit()]
        for i in range(1,len(a)):
            if a[i-1]>=a[i]:
                return False
        return True
